test/utils.js

```diff
@@ -1,16 +1,44 @@
-const {SEO} = require("..");
-const assert = require('assert');
+const { SEO } = require("..");
+const assert = require("assert");
+const { structuredDataTest } = require("structured-data-testing-tool");
+const { Google, Twitter, Facebook } = require("structured-data-testing-tool/presets");
 
 exports.getSeoMetadata = function getSeoMetadata(seoConfig, config, pageType, data, opts) {
-  return new SEO(seoConfig)
-    .getMetaTags(config, pageType, data, opts)
-    .toString();
-}
+  return new SEO(seoConfig).getMetaTags(config, pageType, data, opts).toString();
+};
 
 exports.assertContains = function assertContains(expected, actual) {
   assert(actual.includes(expected), `Expected ${actual} to contain ${expected}`);
-}
+};
 
 exports.assertDoesNotContains = function assertDoesNotContains(expected, actual) {
   assert(!actual.includes(expected), `Expected ${actual} not to contain ${expected}`);
-}
+};
+
+exports.testSchema = async (url, schemas, addPreset = true) => {
+  var result;
+  const obj = {};
+  if (addPreset) {
+    // Check for compliance with Google, Twitter and Facebook recommendations
+    obj.presets = [Google, Twitter, Facebook];
+  }
+  if (schemas.length > 0) {
+    // Check the page includes a specific Schema (see https://schema.org/docs/full.html for a list)
+    obj.schemas = schemas;
+  }
+  return Promise.resolve(structuredDataTest(url, obj))
```

Comment:
no need to wrap `structuredDataTest` with `Promise.rsolve`

Reply:
somehow it's taking a long time to respond when I remove `Promise.resolve`.